pandas/tests/arrays/test_array.py

```diff
@@ -254,3 +255,45 @@ def test_array_not_registered(registry_without_decimal):
     result = pd.array(data, dtype=DecimalDtype)
     expected = DecimalArray._from_sequence(data)
     tm.assert_equal(result, expected)
+
+
+class TestArrayAnalytics(object):
```

Comment:
there are already tests in pandas/tests/extension/base/methods.py are these supplmental?

Reply:
Extension arrays and PandasArrays do not have the same interface, so the tests in pandas/tests/extension/base/methods.py will not work for PandasArrays. E.g. ``.value_counts``, ``.count`` etc. are not found in PandasArrays (or numpy arrays).

BTW, I copied these tests from pandas/tests/series/test_analytics.py.